root/static/lib/flot/jquery.flot.js

```diff
@@ -34,10 +34,6 @@ Licensed under the MIT license.
 // the actual Flot code
 (function($) {
 
-	// Cache the prototype hasOwnProperty for faster access
-
-	var hasOwnProperty = Object.prototype.hasOwnProperty;
```

Comment:
Does ESLint actually complain about files under root/static/lib/? (I would avoid touching these in general, not that the changes seem problematic.)

Reply:
I don't remember if it complains, IIRC we just decided that `hasOwn` seemed better so I made the change everywhere...